apps/web/pages/_app.tsx

```diff
@@ -58,11 +66,10 @@ const config = createConfig({
   },
   ssr: true,
 });
-
 const queryClient = new QueryClient();
 const sprigEnvironmentId = process.env.NEXT_PUBLIC_SPRIG_ENVIRONMENT_ID;
 
-export type NextPageWithLayout<P = unknown, IP = P> = NextPage<P, IP> & {
+export type NextPageWithLayout<P = object, IP = P> = NextPage<P, IP> & {
```

Comment:
is this change necessary? (TS usually complains about `object`)

Reply:
It complained about keeping it {}. Will change it to unknown